### PR TITLE
Create 2017-09-26-release-cadence.md

### PR DESCRIPTION
**Pull request by dslomov**

### CODE REVIEW

**Comment by dslomov** [comment]
Do not review yet

File: _posts/2017-09-27-release-cadence.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+   (0.7.0rc1, 0.7.0rc2 and so on), and release 0._minor_.0 version at the end of baking period.
+ * If critical bugs are discovered after the release, we will fix them and issue *patch releases*
+   as needed (0.7.1, 0.7.2 and so on). Patch releases are always patches on top of existing minor 
+   releases - they are never cut from HEAD.
+ * No new features or breaking changes ever appear in patch releases, and no new features or breaking
+   changes are added to release candidates after they have been cut.
+   
+Our website has more details on [release policy](https://bazel.build/support.html#releases).
+
+As a result of this change, we now issue one minor release per month (0.7 in October 2017, 0.8 in November 2017 and so on).
```
Comment (damienmg) [inline _posts/2017-09-27-release-cadence.md]: Repetition, migth want to remove it (the first dot is strong enough from my point of view)

**Comment by damienmg** [comment]
Merge?

**Comment by dslomov** [comment]
Let's merge after release policy and roadmap are published.

**Comment by dslomov** [comment]
Oh I see you have merged; pressing the button now.

**Comment by damienmg** [comment]
And everything is pushed now :)

**Pull request merged**